mlir/lib/Transforms/EliminateExplicitRounding.cpp

```diff
@@ -0,0 +1,85 @@
+//===- EliminateExplicitRounding.cpp - Remove redundant extf/truncf pairs -===//
```

Comment:
Looks like this file should be in `Dialect/Arith/Transforms`.

Reply:
Good idea! I will move it to `Dialect/Arith/Transforms`